chcemvediet/static/main/css/custom.sass

```diff
@@ -80,6 +80,50 @@ a:hover, a:focus
       padding-top: chv_baseline_fix($chv-fontsize-base, 1)
       overflow: hidden
 
+//
+// Casestudy link component
+//
+
+.chv-casestudy.chv-casestudy
+  display: block
+  padding: ($chv-baseline - 1px)
+  margin-bottom: ($chv-baseline * 1.2)
+  border: solid 1px $chv-color-gray-ll
+  border-radius: 4px
+  color: #4d4d4d
+  &:hover, &:focus
+    color: darken(#4d4d4d, 20%)
+  @include clearfix()
+  > .chv-casestudy-img
+    display: block
+    float: left
+    height: ($chv-baseline * 8)
+    width: ($chv-baseline * 8)
+    margin-right: $chv-baseline
+    margin-bottom: ($chv-baseline * 1.5)
```

Comment:
No spravil som na to hack, ale riesenie vo vseobecnosti to nie je dobre, ale je to velmi jednoduche. Stacilo pridat margin-bottom pre `img`, aby sa tam ten text nezmestil. Vo vseobecnosti to ale nie je dobre riesenie, v nasom pripade to riesi situaciu. Inak obtekanie textu nastava aj pri blogs itemoch.

Reply:
Obtekanie textu by sa zislo spravit tak, ze:

a) Pri sirokom boxe pre text je text vzdy napravo od obrazku. Teda ak je text dlhsi nez obrazok, tak stale pojde v tlpci napravo od neho. 

b) Pri uzkom boxe je text vzdy pod obrazkom. Pricom datum moze asi ostat vedla obrazka (ak vojde).

Reply:
Pozor, tento vacsi margin sposobuje, ze pod obrazkom je prilis velka medzera. Takze to vyzera nesymetricky:

![margin](https://user-images.githubusercontent.com/5547275/116815007-d93d9f00-ab5b-11eb-94a4-e3aa5791428f.png)

Reply:
Datum sa zmesti aj vedla obrazka, ale rozsah v ktorom sa zmesti vedla je velmi maly, preto som ho zalomil pod obrazok spolu s textom. Mozeme to vyuzit pri blog componente, tam je mensi obrazok a tym padom viacej miesta pre datum.

Reply:
Ok. Kludne moze byt datum pod obrazkom pri reze, kde je aj text pod obrazkom.